### PR TITLE
Valid form

### PR DESCRIPTION
**Pull request by ahmadsarfrazx**
- Added a modal using javascript.
- Created an object for the validation of the contact form.
- Made required changes  form is working now

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    e.preventDefault();
+    msgError.classList.toggle('error');
+
+    setTimeout(() => { msgError.classList.toggle('error'); }, 7000);
+  } else {
+    msgSuccess.classList.toggle('success');
+
+    setTimeout(() => { form.submit(); }, 6000);
+  }
+});
```
Comment (franclobo) [inline index.js:140]: - [ ] When I clicked on the submit button the error 403 appeared, please 🙏🏽 fix it for a better user experience.

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @ahmadsarfrazx, 

# Required Changes ♻️
Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] **[OPTIONAL]** ⚠️ Kindly review the announcements in the Slack channel they informed us about a change in the [README template](https://github.com/microverseinc/readme-template) **IT IS GOING TO BE MANDATORY NEXT WEEK** ⚠️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmadsarfrazx** [comment]
- Made required changes  form is working now

File: index.js (hunk 1 above)
Comment (Owusu-Desmond) [inline index.js:140]: - As requested by the previous reviewer,  When the form is submitted the redirected page returned error 403 (Forbidden) which is not expected,
- I think the expected page is this :arrow_down: 
Screenshot from 2022-11-19 14-57-28
- But as you can see from the video below your page does not return a page that looks like the above image.  Please 🙏🏽 fix it for a better user experience.

[Screencast from 11-19-2022 02:52:03 PM.webm](https://user-images.githubusercontent.com/92303891/202856872-f2797bf6-c461-4ab0-ade7-fc1b80c5f58a.webm)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @ahmadsarfrazx and @franclobo  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] warning Kindly review the announcements in the Slack channel they informed us about a change in the [README template](https://github.com/microverseinc/readme-template) IT IS GOING TO BE MANDATORY NEXT WEEK warning

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmadsarfrazx** [comment]
Hello sir, 

I have tried submitting this form but it is working in my side perfectly.

**Comment by ahmadsarfrazx** [comment]
Hi, @Owusu-Desmond the form submission is working perfectly on my side.

**Comment by Owusu-Desmond** [comment]
Hi @ahmadsarfrazx 

Yes it works, I have tried it again and it is working, Please am very sorry about that
Please submit for review again and it will be approved

Thanks for letting me know about that :pray:

**Comment by ahmadsarfrazx** [comment]
its ok. thank you

**Review by Santiago220991 — APPROVED** [review]
Hi @ahmadsarfrazx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark:  Applied changes suggested by the last reviewer,
:heavy_check_mark: No linter errors.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**